### PR TITLE
build: add stable env configuration file

### PR DESCRIPTION
**Pull request by ds-asmierzchalski**
## Description

A new values.yaml for the stable configuration was added similar to values-int.yaml.

## Why

With the new stable environment a specific values.yaml is necessary for the deployment of the service. 

## Issue

Relates to https://github.com/eclipse-tractusx/portal/issues/408

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/policy-hub/blob/main/docs/technical-documentation/dev-process/How%20to%20contribute.md)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added copyright and license headers, footers (for .md files) or files (for images)

### CODE REVIEW

**Comment by MaximilianHauer** [comment]
@ds-asmierzchalski  can you please add some info ?

**Comment by ds-asmierzchalski** [comment]
Hi @MaximilianHauer I updated the description. Sorry for the inconvenience.

File: environments/argocd-app-templates/appsetup-stable.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+###############################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (evegufy) [inline environments/argocd-app-templates/appsetup-stable.yaml:2]: please change to current year, as the file was just created

File: environments/helm-values/values-stable.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+###############################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (evegufy) [inline environments/helm-values/values-stable.yaml:2]: please change to current year, as the file was just created

**Review by evegufy — CHANGES_REQUESTED** [review]
I think it makes sense to direct the PR towards main directly not https://github.com/eclipse-tractusx/policy-hub/tree/feature/add-stable-config as that branch is essentially just a couple commits behind main and not checked out from the 1.1.0 version tag, if that was the assumption.

**Comment by ds-asmierzchalski** [comment]
Updated the pull request to go against main.

File: environments/argocd-app-templates/appsetup-stable.yaml (hunk 1 above)
Comment (ds-asmierzchalski) [reply environments/argocd-app-templates/appsetup-stable.yaml]: the year was updated

File: environments/helm-values/values-stable.yaml (hunk 2 above)
Comment (ds-asmierzchalski) [reply environments/helm-values/values-stable.yaml]: the year was updated

**Review by evegufy — APPROVED** [review]

**Pull request merged**